client/src/components/trip-events-container/TripEventsContainer.jsx

```diff
@@ -12,10 +12,10 @@ import globeWithMarkers from "../../assets/globeWithMarkers.png";
 
 export default function TripEventsContainer({ data, tripID = 0 }) {
   let navigate = useNavigate();
-
-  const trip = data[tripID - 1];
-
-  const events = trip.events.map((event) => {
+  const trip = data.filter((trip) => {
+    if (trip.id === tripID) return trip;
+  });
+  const events = trip[0].events.map((event) => {
```

Comment:
if it is guaranteed to find a matching trip, you can do 
data.filter((trip) =>).map(({events}) =>)
instead of using trip[0]

Reply:
Its crushes. I know its not looks pretty, but it works. I can filter the data at AppContainer and inject to EventListComp the correct trip/events only, maybe after we finish the min requirements for the app.